composer.json

```diff
@@ -110,7 +110,7 @@
             "@phpstan",
             "@psalm"
         ],
-        "stan-setup": "cp composer.json composer.backup && composer require --dev phpstan/phpstan-shim:^0.11.18 psalm/phar:^3.5 && mv composer.backup composer.json"
+        "stan-setup": "cp composer.json composer.backup && composer require --dev phpstan/phpstan:^0.12.0 psalm/phar:^3.7 && mv composer.backup composer.json"
```

Comment:
Can we just require-dev it now?

Reply:
I don't expect the average contributor to run pslam/phpstan so I would rather not add extra dependencies.

People like me who use them for multiple libs/repos have it installed globally and having them installed per project is just wasteful.

Reply:
bizarre but ok.